samples/bluetooth/peripheral_hids_mouse/README.rst

```diff
@@ -100,11 +100,15 @@ The HID service specification does not require encryption (:kconfig:option:`CONF
 Building and running
 ********************
 
-To build this sample with the :ref:`nrf_rpc_ipc_readme` library on the nRF5340 DK, set the :makevar:`EXTRA_CONF_FILE` option to the :file:`overlay-nrf_rpc.conf` file.
+To build this sample with the :ref:`ble_rpc` library, add the following parameters:
+
+* set the :makevar:`SNIPPET` option to ``nordic-bt-rpc``,
+* set the :makevar:`FILE_SUFFIX` option to ``bt_rpc``,
+* set the :makevar:`EXTRA_CONF_FILE` option to the :file:`overlay-nrf_rpc.conf` file.
```

Comment:
Why do we need extra conf file here? Using file suffix should be suitable

Reply:
Talking to @dchat-nordic I decided to remove the overlay config, so modifying the sample as suggested in this comment.